Source/VCGeneration/SplitAndVerifyWorker.cs

```diff
@@ -105,6 +105,7 @@ async Task DoWork(Split split)
       }
       catch {
         split.ReleaseChecker();
+        throw; // This can be better handled farther up the call stack.
```

Comment:
Nitpick: I don't think the comment is necessary. The catch being here only to release the checker makes sense to me.